<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>830</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1110</width>
    <height>830</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1110</width>
    <height>830</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #00BFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 26px; color: #FFD700;</string>
    </property>
    <property name="text">
     <string>Ru</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">language_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 26px; color: #FFD700;</string>
    </property>
    <property name="text">
     <string>Eng</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">language_group</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="form_text_button">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gold;
    background: none;
    color: gold;
	font-size: 16px;
	font-family: 'gilroy';
	font-weight: 900;
}
QPushButton:hover {
    background: gold;
    color: white;
} </string>
    </property>
    <property name="text">
     <string>Сформировать текст</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold;
font-size: 16px; font-weight: 900;</string>
    </property>
    <property name="text">
     <string>Количество слов:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px; color: gold;</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">count_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px; color: gold;</string>
    </property>
    <property name="text">
     <string>50</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">count_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px; color: gold;</string>
    </property>
    <property name="text">
     <string>35</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">count_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px; color: gold;</string>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">count_group</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="text_field">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>981</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: 00BFFF;
border: 2px solid gold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>680</y>
      <width>440</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white; border: 1px solid gold;</string>
    </property>
   </widget>
   <widget class="QLabel" name="timer_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>680</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px; border:  1px solid gold; color: gold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: gold; 
	border: 1px solid gold;
	font-weight: bold;
}

QPushButton:hover {
    background: gold;
    color: white;
} </string>
    </property>
    <property name="text">
     <string>Главное меню</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>110</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gold; font-weight: bold; font-size: 14px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="language_group"/>
  <buttongroup name="count_group"/>
 </buttongroups>
</ui>
